### PR TITLE
Remove more facilitator code

### PR DESCRIPTION
**Pull request by megcrenshaw**
Found some more facilitator code mentions on the frontend as I was doing work –– this removes FitWeekend and TeacherCon from DetailView and updates a comment.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx [hunk 1]
```diff
@@ -692,36 +608,6 @@ export class DetailViewContents extends React.Component {
     );
   };
 
-  renderRegistrationLinks = () => {
```
Comment (megcrenshaw) [inline apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx:695]: This entire method wasn't being used –– see https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx for original file

Comment (TurnerRiley) [reply apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx:695]: Nice catch!

**Review by TurnerRiley — APPROVED** [review]
This looks great! Thank you for your continued work on removing all this!

**Comment by davidsbailey** [comment]
woohoo!

**Pull request merged**